snippets/distr-tracing-tempo-tempomonolithic-custom-resource.adoc

```diff
@@ -3,41 +3,54 @@
 //
 // * modules/distr-tracing-tempo-install-tempomonolithic-web-console.adoc
 // * modules/distr-tracing-tempo-install-tempomonolithic-cli.adoc
-The following `TempoMonolithic` CR creates a TempoMonolithic deployment with trace ingestion over OTLP/gRPC and OTLP/HTTP, storing traces in a supported type of storage and exposing Jaeger UI via a route:
+. Example `TempoMonolithic` CR
```

Comment:
@rubenvp8510, do we have everything that we need here?
I'm looking at this task in the [Jira](https://issues.redhat.com/browse/OBSDOCS-1676)
```The OTEL deployment should include all recommended and required components - RED metrics, k8sattributes processor```